luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -41,15 +41,12 @@ local powerThresholds = {
     {techLevel = 4.5, threshold = 725000}
 }
 
+
+local allyID = select(6, Spring.GetTeamInfo(teamID))
```

Comment:
Why was this moved outside the loop? `teamID` is undefined here.

Reply:
yep, fixed that just as the review came in